py/desiutil/names.py

```diff
@@ -0,0 +1,62 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+"""
+============
```

Comment:
The number of `=` should match the length of the title.

Reply:
Done